### DISCUSSION TITLE
nuxt generate nitro 500 error

### DISCUSSION THREAD

**Discussion by DenFin** [nuxt/nuxt#23631] [Questions]
Hi there,

we deploy our static nuxt site on Netlify. The site just works fine but in the deploy logs we get a lot of 500 errors:
<img width="638" alt="Bildschirmfoto 2023-10-12 um 08 29 01" src="https://github.com/nuxt/nuxt/assets/11504119/165361d6-ac41-4d66-b90c-61185c205f8a">

I tried setting the logLevel in nuxt.config like this, but it didn't help:

```
  nitro: {
    logLevel: +999,
  },
```

How can I get Nitro to log more information on the error?

**Comment by nachoadjust** [comment]
Similar situation here, but for Vercel (and also when running `nuxt build` locally).

We know we have a problem where prerender routes throw 500, but we have no clue where the problem can be coming from.

The failing routes also change from build to build, so we cannot attribute it to a specific page or plugin lifecycle. 

Having more info in the 500 logs would help a lot!!

**Reply by nachoadjust** [reply]
Update:
I've been able to trace errors with messages other than 500s.

In my case the project I am working with uses Storyblok. 
After reading some of their current issues I found [this fix](https://github.com/storyblok/storyblok-nuxt/issues/557#issuecomment-1758105858) that adds better log messages.

It's by no means a fix to the original problem, but maybe it helps someone else. 

<img width="1277" alt="Screenshot 2023-11-14 at 15 43 29" src="https://github.com/nuxt/nuxt/assets/40995655/4413922c-414f-48fe-98c1-53f2cdce3f2c">

Note: 
The image shows logs for Nuxt 3.6.5 (version I am using), but I also got a similar result when testing for 3.8.1

**Comment by steffenstolze** [comment]
@DenFin @nachoadjust any more ideas about this?

I'm experiencing the same with Nuxt 3 SSG and Directus.

I assume that it's somehow related with @nuxtjs/i18n - dunno why, just a feeling.

I was able to mitigate the 500 errors by changing the nitro config in `nuxt.config.ts`

```ts
nitro: {
    debug: true,
    prerender: {
      crawlLinks: true,
      concurrency: 1,
    },
  },
```

Funny enough.. I wanted to try the latest feature:

```ts
experimental: {
    sharedPrerenderData: true,
},
```

Then I see 500s again :)
